src/pages/docs/chat/react-ui-components/providers.mdx

```diff
@@ -0,0 +1,556 @@
+---
+title: "Providers and Hooks"
+meta_description: "Comprehensive documentation for Ably Chat React UI Components providers and hooks"
+meta_keywords: "Ably, realtime, Chat UI, React, providers, hooks, context"
+---
+
+This page documents the providers and hooks available in the Ably Chat React UI Components library. These providers and hooks are used to manage state and provide functionality to the components.
+
+## Provider Setup <a id="provider-setup"/>
+
+The Ably Chat React UI Components rely on several React context providers to function properly. The `ChatClientProvider` is the core provider that manages access to the Ably Chat client, and should be placed at the root of your application.
+
+Other providers like `ThemeProvider`, `AvatarProvider`, and `ChatSettingsProvider` can be used to manage themes, avatars, and chat settings respectively. These providers should be placed at the highest level of any component tree that contains Chat UI components.
```

Comment:
I think `ChatSettingsProvider` should come above the other two (as its arguably the most important)